scheduler/scheduler-smartproxy-common/src/main/java/org/ow2/proactive/scheduler/smartproxy/common/AbstractSmartProxy.java

```diff
@@ -313,15 +307,25 @@ public JobId submit(TaskFlowJob job, String localInputFolderPath, String pushUrl
             throws Exception, SubmissionClosedException, JobCreationException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)